app/scripts/lib/constants.js

```diff
@@ -77,7 +77,10 @@ define([], function () {
     ACCESS_TYPE_ONLINE: 'online',
     ACCESS_TYPE_OFFLINE: 'offline',
 
-    DEFAULT_XHR_TIMEOUT_MS: 2500
+    DEFAULT_XHR_TIMEOUT_MS: 2500,
+
+    SIGNIN_SUCCESS: 'signin.success',
```

Comment:
Since these are both messages can you add a prefix or suffix saying so? I wonder if there is a better place for these, closer to where they are used so some local context exists. I sometimes wonder if constants.js was a mistake.